scapy/layers/tls/automaton.py

```diff
@@ -219,5 +219,5 @@ def flush_records(self):
 
     def vprint(self, s=""):
         if self.verbose:
-            logging.info("> %s" % s)
+            log_interactive.info("> %s" % s)
```

Comment:
You don't have to format the string yourself when using the `logging` functions (so that the log is not formatted if it is not displayer):
```Python
log_interactive.info("> %s", s)
```